src/server/js/schema.js

```diff
@@ -1,14 +1,39 @@
 import { merge } from 'lodash';
 import { makeExecutableSchema } from 'graphql-tools';
 
-import {
-    typeDefs as Health,
-    resolvers as healthResolvers,
-} from './resources/health/schema';
+import HouseModel from './resources/houses/model';
+import * as houseSchema from './resources/houses/schema';
+import UserModel from './resources/users/model';
+import * as userSchema from './resources/users/schema';
+
+const queryTypeDefs = `
+    type Query {
+        allHouses: [House],
+        housesByAdministrator(administratorId: ID!): [House],
+        allUsers: [User],
+        userByEmail(email: String): User,
+    }
+`;
```

Comment:
ideally i would be able to define these model-specific queries in the model's own `typeDefs` and move the associated resolvers (below) there. alas, that could not be done - i cannot define multiple `type Query`s separately and merge them. (there is a third-party package that lets you do that,  if this gets messy)